packages/manager/modules/pci/src/projects/project/failover-ips/order/legacy/order.html

```diff
@@ -37,19 +37,11 @@
                 data-on-change="$ctrl.onRegionChange(modelValue)">
                 <span data-ng-bind="$item.name"></span>
             </oui-select>
-            <div class="oui-field__error" data-ng-class="{
-                'd-block': $ctrl.regionIsInvalid
-            }">
-                <span role="alert">
-                    <span class="oui-icon oui-icon-error" aria-hidden="true"></span>
-                    <span data-translate="pci_projects_project_failoverip_order_legacy_region_invalid"></span>
```

Comment:
Could be removed from `packages/manager/modules/pci/src/projects/project/failover-ips/translations/Messages_fr_FR.json` as well.